plugins/tabletools/dialogs/tableCell.js

```diff
@@ -426,7 +420,28 @@ CKEDITOR.dialog.add( 'cellProperties', function( editor ) {
 					} ]
 				} ]
 			} ]
-		} ],
+		}, {
+      id: 'padding',
+      label: langTable.cellPad,
+      elements: [ {
+          type: 'vbox',
+          padding: 0,
+          children: [
+            { type: 'hbox', padding: 0, widths: [ '33%', '34%', '33%' ], children: [ 
+              spacer, 
+              lengthUIElement( 'padding', 'padding-top', langCell.paddingTop, langTable.invalidCellPadding, 'vbox'), 
+              spacer ] },
+            { type: 'hbox', padding: 0, widths: [ '33%', '34%', '33%' ], children: [ 
+              lengthUIElement( 'padding', 'padding-left', langCell.paddingLeft, langTable.invalidCellPadding, 'vbox'), 
```

Comment:
Code style: use space after first and before last parenthesis if the function call has one or more arguments.